tests/test_slurm.py

```diff
@@ -32,7 +33,7 @@ def setUpClass(cls):
                 # output a job id
                 while [[ "$1" == -* ]] ; do shift ; done
                 export SLURMD_NODENAME={node}
-                source $@ > slurm-12345.out 2>&1
+                source $@
```

Comment:
Why did you change this? It was mimicking the behavior of sbatch, producing a slrum-<jobid>.out file